docs/aws-quickstart.md

```diff
@@ -0,0 +1,51 @@
+# Register inside Google Cloud Platform
+
+## Create New Access Key
+
+[Create New Access Key](https://console.aws.amazon.com/iam/home?region=us-east-2#/security_credentials)
+
+Put key ID and secret into environment variables:
+* NSM_AWS_ACCESS_KEY_ID
+* NSM_AWS_SECRET_ACCESS_KEY
+
+# Prerequisites
+
+## Setup AWS.
+
+```shell
+* make aws-init
+```
+
+## Use AWS environment.
+
+Before start using of AWS, please execute the following command:
+
+```shell
+export CLUSTER_RULES_PREFIX=gke
```

Comment:
Can we have `.env/aws.env` similar to [.env/gke.env](https://github.com/networkservicemesh/networkservicemesh/blob/master/.env/gke.env)?